runtimes/shared-configuration/src/assets.rs

```diff
@@ -32,6 +32,7 @@ parameter_types! {
 	pub const MetadataDepositBase: Balance = deposit(1, 68);
 	pub const MetadataDepositPerByte: Balance = deposit(0, 1);
 	pub const AssetAccountDeposit: Balance = deposit(1, 18);
+	pub const ZeroAssetAccountDeposit: Balance = 0;
```

Comment:
We should also change the `MetadataDepositBase` to 0. Especially since you did it in the mock